server/src/routes/api/v1/parksRouter.js

```diff
@@ -12,4 +12,14 @@ parksRouter.get("/", async(req, res) => {
   }
 })
 
+parksRouter.get("/:id", async (req,res) => {
+const parkId = req.params.id;
+  try {
+    const park = await Park.query().findById(parkId);
+    return res.status(200).json({ park: park  })
+  } catch (error) {
+    return res.status(500).json({ errors: error})
+  }
+})
+
```

Comment:
lines 16 and 17 should be inline with one another - all you need to do here is indent line 16